sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/zapwrapper/helper/ZapProductMessageHelper.java

```diff
@@ -81,7 +92,7 @@ private void writeProductErrorForExitCode(ZapWrapperExitCode exitCode) throws IO
             break;
         case API_DEFINITION_CONFIG_INVALID:
             productMessageSupport.writeMessage(new SecHubMessage(SecHubMessageType.ERROR,
-                    "Only a single API file can be provided. Please use a single file for the API definition inside the filesystem->files section of the SecHub configuration."));
+                    "Please use files instead of folders for the API definition with the filesystem->files section of the SecHub configuration."));
```

Comment:
The meaning has complete changed.
1. is it correct that you can add multiple files?
2. API_DEFINITION_CONFIG_INVALID: is the message always correct for the case?

Reply:
Yes in the first version we supported only a single file, but currently we can handle multiple files, but folders are not supported inside the data section. I forgot to update the message the last time and saw the old user message while I was working on the current issue. It is documented with the examples for any user, that they can use multiple files. I will update the message to a more generic version so the user knows which section to check.